pyproject.toml

```diff
@@ -17,6 +17,11 @@ xarray = "^0.21.1"
 numba = "^0.55.1"
 threadpoolctl = "^3.1.0"
 tqdm = "^4.62.3"
```

Comment:
That's ok, but next time make another PR if two different issues are addressed, e.g. addind new dependency and creating documentation.
It's easier for review ;)

Reply:
Right. Most of the dependencies added are for documentation purposes. I saw that you included matplotlib in the other PR so I think it can be removed from here.